src/main/java/org/folio/linked/data/mapper/dto/monograph/common/NoteMapper.java

```diff
@@ -28,16 +28,16 @@ public List<NoteDto> toNotes(JsonNode doc, Set<PropertyDictionary> notePropertie
       .flatMap(entry ->
         stream(spliteratorUnknownSize(entry.getValue().iterator(), ORDERED), false)
           .map(value -> {
-            var noteDto = new NoteDto();
-            noteDto.setValue(List.of(value.textValue()));
-            noteDto.setType(List.of(entry.getKey()));
-            return noteDto;
+            var notes = new Note();
```

Comment:
var `note`?

as I remember our dto has methods set with return 'this' and it can be like
```
return newNodeDto
    .setValue(List.of(value.textValue()))
    .setType(List.of(entry.getKey()));
```

Reply:
correct, fixed, thx!